.github/workflows/tests.yml

```diff
@@ -3,11 +3,12 @@ on: [push]
 
 jobs:
   test:
-    name: Node.js ${{ matrix.node_version }}
+    name: Node.js ${{ matrix.node_version }} & Experimental ${{ matrix.experimental }}
```

Comment:
What is the experimental matrix for?

Reply:
I'm just thought I'd run the 2 experiments in parallel instead of sequence but I'm happy to change it back if you prefer the original setup. I just realized that with this setup I'm not running `xo` in CI which is not great. Should I restore the original script that uses `npm test` or should I add `xo` as a step here?

Reply:
I would prefer just a single job with `npm test`, but I'll leave it to @vadimdemedes to  decide.

Reply:
Restored.